abixen-platform-core/src/main/resources/liquibase/changelog/comment-changelog.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd"
+        logicalFilePath="liquibase/changelog/comment-changelog.xml">
+
+    <changeSet author="MariuszKumor" id="comment-v1">
+        <createTable tableName="comment">
+            <column name="id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="message" type="varchar(40)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="created_date" type="timestamp"/>
+            <column name="last_modified_date" type="timestamp"/>
+            <column name="created_by_id" type="bigint"/>
+            <column name="last_modified_by_id" type="bigint"/>
+            <column name="layout_id" type="bigint">
```

Comment:
@mariuszkumor 

```
<column name="created_date" type="timestamp"/>
<column name="last_modified_date" type="timestamp"/>
<column name="created_by_id" type="bigint"/>
<column name="last_modified_by_id" type="bigint"/>
<column name="layout_id" type="bigint">
```

do we need above fields in comment model?
 
Because , Y i am skiing this question is  all above fields are inherited form AuditingModel.  So we need to keep that fields. if we agreed to don't keep those fields in comment-chagelog.xml . i will not inhered the AuditingModel class.